examples/model_selection/grid_search_digits.py

```diff
@@ -60,11 +60,11 @@
     print()
     print("Grid scores on development set:")
     print()
-    means = clf.results_['test_mean_score']
-    stds = clf.results_['test_std_score']
-    for i in range(len(clf.results_['params'])):
+    means = clf.cv_results_['mean_test_score']
+    stds = clf.cv_results_['std_test_score']
+    for mean, std, params in zip(means, stds, clf.cv_results_['params']):
```

Comment:
I used this chance to address an [old review](https://github.com/scikit-learn/scikit-learn/pull/6697#discussion_r67387474) ;)